specification/maps/data-plane/Common/preview/1.0/common.json

```diff
@@ -1,712 +1,712 @@
-      "in": "query",
-      "description": "Y coordinate of the tile on zoom grid. Value must be in the range [0, 2<sup>`zoom`</sup> -1].\n\nPlease see [Zoom Levels and Tile Grid](https://docs.microsoft.com/en-us/azure/location-based-services/zoom-levels-and-tile-grid) for details.",
-      "required": true,
-      "type": "integer",
-      "format": "int32",
-      "x-ms-parameter-location": "method"
-    }
-  },
-  "responses": {
-    "200AsyncV2": {
-      "description": "The operation is running or complete. If the operation was successful, use the Resource-Location header to obtain the path to the result.",
-      "schema": {
-        "$ref": "#/definitions/LongRunningOperationResult"
-      },
-      "headers": {
-        "Resource-Location": {
-          "type": "string",
-          "description": "If successful, a URI where details on the newly created resource can be found."
-        }
+      "GeoJsonFeatureCollectionData": {
+        "type": "object",
+        "properties": {
+          "features": {
+            "description": "Contains a list of valid `GeoJSON Feature` objects.",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/GeoJsonFeature"
+            }
+          }
+        },
+        "required": [
+          "features"
+        ]
+      },
+      "Ontology": {
+        "type": "string",
+        "description": "The ontology version of this dataset.",
+        "readOnly": true
       }
     },
-    "202Async": {
-      "description": "**Supported only for async request.**\nRequest Accepted: The request has been accepted for processing. Please use the URL in the Location Header to retry or access the results.",
-      "headers": {
-        "Location": {
-          "type": "string",
-          "description": "New URL to check for the results of the long running process."
-        }
+    "parameters": {
+      "ClientId": {
+        "name": "x-ms-client-id",
+        "description": "Specifies which account is intended for usage in conjunction with the Azure AD security model.  It represents a unique ID for the Azure Maps account and can be retrieved from the Azure Maps management  plane Account API. To use Azure AD security in Azure Maps see the following [articles](https://aka.ms/amauthdetails) for guidance.",
+        "type": "string",
+        "in": "header",
+        "x-ms-parameter-location": "client"
+      },
+      "Language": {
+        "name": "language",
+        "in": "query",
+        "description": "Language in which search results should be returned. Should be one of supported IETF language tags, case insensitive. When data in specified language is not available for a specific field, default language is used.\n\nPlease refer to [Supported Languages](https://docs.microsoft.com/en-us/azure/azure-maps/supported-languages) for details.",
+        "type": "string",
+        "x-ms-parameter-location": "method"
+      },
+      "ResponseFormat": {
```

Comment:
`ResponseFormat` and `JsonFormat` are both used to describe... response format. Perhaps this should simply be `JsonFormat` and have `x-ms-client-default` and `TextFormat` (which seems highly inaccurate) becomes `ResponseFormat`.

Reply:
Rename the parameters as suggested ([Commit](https://github.com/Azure/azure-rest-api-specs/pull/16030/commits/f88b0b1123e46fe03532092488463d4139503982))